### PR TITLE
Op 988 fix redirect access method

### PR DESCRIPTION
**Pull request by dk-WZFinTech**

### CODE REVIEW

File: figo/models.py [hunk 1]
```diff
@@ -796,14 +869,19 @@ class Challenge(ModelBase):
...
+    ]
 
     id = None
     title = None
     label = None
     format = None
     data = None
     type = None
+    location = None
+    created_at = None
```
Comment (dk-WZFinTech) [inline figo/models.py:884]: This is main fix: 

    __dump_attributes__ = [
        "id", "title", "label", "format", "data", "type", "location",
        "created_at"
    ]

    id = None
    title = None
    label = None
    format = None
    data = None
    type = None
    location = None
    created_at = None

all other staff is clean up code

Comment (piotrlukaj) [reply figo/models.py:884]: Looks ok! A lot changed because of `black` 👍

**Review by piotrlukaj — APPROVED** [review]
It looks ok! A lot of code was changed as I see, but thankfully black handled this 👍

**Comment by dk-WZFinTech** [comment]
> ... but thankfully black handled this +1

Actually I did it by myself without black :P I was scary it will find to much :)

**Review by escer — APPROVED** [review]

**Pull request merged**